bin/resources/nvrtc_cli.cpp

```diff
@@ -0,0 +1,574 @@
+// Copyright (c) 2022, Compiler Explorer Authors
```

Comment:
Sounds like a plan, we can arrange it to be built properly instead of having the compile code in here. Might be a good idea to add a target for it in the Makefile to build this specifically that we then can invoke.

Reply:
I've created a PR for it: https://github.com/NVIDIA/jitify/pull/105

I've tested it with the below code in `cuda.yaml` and it works. Are there potentially better ways to set up the build?

```check_exe: bin/nvcc --version && bin/nvrtc_cli --version
    depends:
      - compilers/c++/x86/gcc 10.2.0
    fetch:
      - https://developer.download.nvidia.com/compute/cuda/{name}/local_installers/cuda_{name}_{build}_linux.run combined.sh
      - https://raw.githubusercontent.com/NVIDIA/jitify/6bc6e25e13300a8bc7520ed3bf977865201183b7/nvrtc_cli.cpp nvrtc_cli.cpp
    script: |
      mkdir -p cuda/{name}
      sh combined.sh --silent --toolkit --toolkitpath=$(pwd)/cuda/{name}
      /opt/compiler-explorer/gcc-10.2.0/bin/g++ nvrtc_cli.cpp -o cuda/{name}/bin/nvrtc_cli -O3 -Wall -Wextra -std=c++11 -Icuda/{name}/include -Lcuda/{name}/lib64 -lnvrtc -Wl,--disable-new-dtags,-rpath,'$ORIGIN/../lib64'
```

Reply:
Thanks all! My feeling is that we're probably OK with the setup as-is for now -- I don't want to hold up any longer! That looks good @benbarsdell  - sorry for the slowness on my part. It's been a tough week at `$dayjob` but hopefully we're out of the woods now. There's a long weekend coming too. So - let's go with it as-is, let me know when the PR is merged upstream and we'll go (though maybe it works without the merged PR?)

Reply:
Looks like we're still waiting on the upstream merge - I'm happy to point at an orphaned commit for now if that gets things going?

Reply:
I think that should be fine. I've updated the branch to reflect the cuda.yaml listed above, and removed the cpp file.